packages/app-backend-vue2/src/components/tree.ts

```diff
@@ -270,7 +273,7 @@ async function capture (instance, index?: number, list?: any[]): Promise<Compone
       backgroundColor: 0xff8344
     })
   }
-  return api.visitComponentTree(
+  return await api.visitComponentTree(
```

Comment:
Await in front of return is unnecessary

Reply:
Resolved in 93f94fda5dc5a780c53fe57112f2bcdb9b1360f6